### PR TITLE
Quoted node command to avoid issues with 'Program Files' etc.

### PR DESCRIPTION
**Pull request by SnoopJ**
Enclosing the call to node in the `kiwi` script prevents command failure for executable paths containing whitespace (e.g. `/c/Program Files/nodejs/node`).

Also adds last-spoke ordering for autocompletion, re: #575

### CODE REVIEW

File: client/src/views/channel.js [hunk 1]
```diff
@@ -366,6 +366,12 @@ _kiwi.view.Channel = _kiwi.view.Panel.extend({
         // Generate a hex string from the nick to be used as a CSS class name
         nick_hex = '';
         if (msg.nick) {
+            // we're in a channel and this nick is, too
+            if (this.model.get("members") && this.model.get("members").getByNick(msg.nick)) {
+              // set last_spoke time
+              this.model.get("members").getByNick(msg.nick).set("last_spoke", msg.time.getTime());
```
Comment (prawnsalad) [inline client/src/views/channel.js]: Indentation of 4 spaces should be used here

File: client/src/views/controlbox.js [hunk 2]
```diff
@@ -208,7 +208,7 @@ _kiwi.view.ControlBox = Backbone.View.extend({
         var that = this,
             inp = $(ev.currentTarget),
             inp_val = inp.val(),
-            meta;
+            meta, last_spoke;
```
Comment (prawnsalad) [inline client/src/views/controlbox.js]: Shouldn't the `sortByOrder` declaration be inside the `_.sortByOrder` function below where it's used?

File: client/src/views/controlbox.js [hunk 3]
```diff
@@ -329,13 +329,20 @@ _kiwi.view.ControlBox = Backbone.View.extend({
...
-            autocomplete_list = _.sortBy(autocomplete_list, function (entry) {
-                // Nicks have a .type property of 'nick'
-                return entry.type === 'nick' ?
-                    entry.match[0].toLowerCase() :
-                    entry.toLowerCase();
-            });
+            // Sort first by last_spoke if appropriate, then by nick
+            autocomplete_list = _.sortByOrder(autocomplete_list,
+                [function (entry) {
+                    if (entry.type === 'nick' && _kiwi.global.settings.get('use_last_spoke_ordering')) {
```
Comment (prawnsalad) [inline client/src/views/controlbox.js]: I would change the `_kiwi.global.settings.get('use_last_spoke_ordering')` condition to `_kiwi.global.settings.get('use_last_spoke_ordering') !== false` to make your new ordering the default

File: config.example.js [hunk 4]
```diff
@@ -218,6 +218,7 @@ conf.client = {
         ignore_new_queries: false,
         count_all_activity: false,
         show_autocomplete_slideout: true,
+        use_last_spoke_ordering: false,
```
Comment (prawnsalad) [inline config.example.js]: Lets change this to true to make your new ordering the default

**Comment by SnoopJeDi** [comment]
Does this PR need anything else @prawnsalad?

**Pull request closed without merging**